test/main/errors/index.test.js

```diff
@@ -140,4 +143,23 @@ describe('sentry', () => {
     const reportedEvents = sentEvents.filter(evt => !!evt)
     expect(reportedEvents.length).toBe(5)
   })
+
+    
+  it('should not send event if log reporting is enabled', () => {
```

Comment:
`it('should send events when exception reporting is enabled', () => {`

Reply:
Fixed up